easybuild/toolchains/compiler/nvhpc.py

```diff
@@ -0,0 +1,106 @@
+##
+# Copyright 2015 Bart Oldeman
+#
+# This file is triple-licensed under GPLv2 (see below), MIT, and
+# BSD three-clause licenses.
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Hercules foundation (http://www.herculesstichting.be/in_English)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+Support for NVIDIA HPC SDK ('NVHPC') compilers (nvc, nvc++, nvfortran) as toolchain compilers. NVHPC is the successor of the PGI compilers, on which this file is based upon.
+
+:author: Bart Oldeman (McGill University, Calcul Quebec, Compute Canada)
+:author: Damian Alvarez (Forschungszentrum Juelich GmbH)
+:author: Andreas Herten (Forschungszentrum Juelich GmbH)
+"""
+
+import easybuild.tools.systemtools as systemtools
+from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.toolchain.compiler import Compiler
+
+
+TC_CONSTANT_NVHPC = "NVHPC"
+
+
+class Nvhpc(Compiler):
```

Comment:
@AndiH Doesn't matter much, but `NVHPC` makes more sense here imho.

```suggestion
class NVHPC(Compiler):

Reply:
This came from `pgi.py`:
```
class Pgi(Compiler):
```
I don't mind at all. But I need to propagate this to the other files, right?

Reply:
Yeah, it has to be consistent in `import` statements & such, indeed.

For `pgi.py` it should've been `PGI` I guess (it doesn't matter much in practice, but `Nvhpc` looks weird to me)

Reply:
Alright! I'll change and propagate to the other files when fixing our other comments.

Thanks!